groupActions/group.go

```diff
@@ -21,7 +21,7 @@ type Locker interface {
 type group struct {
 	groupID string
 	members []groupTypes.ActionHandler
-	locker  Locker
+	Locker
```

Comment:
Please put back the locker as a sub-component. 
The reason behind this that this struct will become more fragile to use as anyone who will use it can randomly call Unlock/Lock/RLock/RUnlock causing panics. `group` struct should be able to handle of locking/unlocking of its mutex.